Cue/app/reducers/library.js

```diff
@@ -5,30 +5,53 @@ import type { Deck } from '../api/types';
 
 export type State = {
   decks: ?Array<Deck>;
+  inaccessibleDecks: ?Array<Deck>;
   localChanges: [];
 };
 
 const initialState: State = {
   decks: null,
+  inaccessibleDecks: null,
```

Comment:
I dont think it matters either way? Even if I did set it to an empty array, I would still need to do the check below and create a new array or make a copy